test_data/database_backup/d3mmmfsmi1d2gh_public_users.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO public.users (id, email, password, name, netid) VALUES (1, 'rahulr2@illinois.edu', '13b1918ffc0b4991910ef0929a935874b20bd349e9de3d2044b0d1c93ae439a3', 'Rahul Rameshbabu', 'rahulr2');
+INSERT INTO public.users (id, email, password, name, netid) VALUES (2, 'parisfe2@illinois.edu', '30c952fab122c3f9759f02a6d95c3758b246b4fee239957b2d4fee46e26170c4', 'Jordi Paris Ferrer', 'parisfe2');
```

Comment:
why is everyone stealing my password ¿ ¿¿

Reply:
Lol